debian/ola.udev

```diff
@@ -13,6 +13,9 @@ SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="10cf", ATTRS{idPro
 # udev rules for the DMXControl Projects e.V. Nodle U1
 SUBSYSTEM=="usb|usb_device", ACTION=="add", ATTRS{idVendor}=="16d0", ATTRS{idProduct}=="0830", GROUP="plugdev"
 
+# udev rules for the DMXCreator 512 Basic device
```

Comment:
Can you also open a PR here to patch the pi file: https://github.com/OpenLightingProject/raspberrypi/blob/master/etc/udev/rules.d/10-local.rules